### ISSUE TITLE
Normalization?

### ISSUE THREAD

**Issue by DGladish** [AntonioDeFalco/SCEVAN#91]
Hi!

Just wanted to see if the read counts have to be normalized to account for sequencing depth, or if unnormalized counts are used as input.

Thank you!

**Comment by AntonioDeFalco** [comment]
Hi @DGladish,
Non-normalized counts are used as input since SCEVAN analyses each sample individually and does not take this normalization into account even in the multi-sample analysis, each sample is considered and then the output is compared. We recommend this type of use, do you need to analyze a merged matrix of different sequencing? 

Thanks, Regards.

**Comment by DGladish** [comment]
Nope! I am working on 4 separate samples (cancer scRNA-seq data at diagnosis & relapse, + a few healthy donors)

**Comment by AntonioDeFalco** [comment]
> Another question, does SCEVAN work if the input is a data frame? Or does the data type have to be matrix?

Both are accepted.

Regards
